### ISSUE TITLE
difference with discovery-swarm 

### ISSUE THREAD

**Issue by KrishnaPG** [hyperswarm/hyperswarm#43]
While searching for DHT related projects, came across these two similar projects, even @mafintosh being in the contributors' list for both.

On the surface these both seem to be doing the same, so confused as to which one of these should be used:

- Discovery Swarm https://github.com/mafintosh/discovery-swarm
- Hyper Swarm https://github.com/hyperswarm/hyperswarm

`Discovery Swarm `seem to be based on [bittorent-dht ](https://github.com/webtorrent/bittorrent-dht) at the bottom, while `Hyper Swarm` seem to be based on [DHT-RPC](https://github.com/mafintosh/dht-rpc).

Would really appreciate if some clarity can be thrown on, if these projects are different, and/or which one should we prefer over the other for future-proofing.

**Comment by mafintosh** [comment]
Yea we should add a section to the readme about this.

TL;DR this is the successor to discovery-swarm and is more or less better in everyway
